roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
The `openshift_excluder` is included in `playbooks/common/openshift-cluster` playbook which is included in both deploy and upgrade playbooks.